lib/cim/prefix-iri-provider.ts

```diff
@@ -0,0 +1,18 @@
+import {IriProvider} from "./iri-provider";
+
+const prefix = "pim:";
+
+export class PrefixIriProvider implements IriProvider {
```

Comment:
The name is good, personally I would name it ```PrefixBasedIriProvider``` to more emphasize the fact the that provider is based on the concept of adding/removing a fixed prefix.

In general I do not know it that is actually a good idea - like what if there are two classes with same IRIs, or is the result really a valid IRI ?

Reply:
> like what if there are two classes with same IRIs

`PrefixIriProvider` specifically is a bijection.

Do you mean it is possible to have a CIM resource (= RDF node) representing two distinct PIM resources? Like two classes or a class that is also an attribute? (on a single level)